Resources/Prototypes/Tiles/floors.yml

```
@@ -652,8 +652,8 @@
   id: FloorReinforced
   name: tiles-reinforced-floor
   sprite: /Textures/Tiles/reinforced.png
-  baseTurf: Plating
-  isSubfloor: false
+  baseTurf: Lattice # Mono
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)